src/app/calculator/fans/fan-analysis/fan-analysis-form/plane-data-form/pressure-readings-form/pressure-readings-form.component.ts

```diff
@@ -48,6 +56,70 @@ export class PressureReadingsFormComponent implements OnInit {
     this.updateTraverseDataSubscription.unsubscribe();
   }
 
+  setImportFile(files: FileList) {
+    this.fileReferences = new Array();
+    if (files) {
+      if (files.length !== 0) {
+        let regex3 = /.xlsx$/;
+        for (let index = 0; index < files.length; index++) {
+          if (regex3.test(files[index].name)) {
+            this.fileReferences.push(files[index]);
+          }
+        }
+        if (this.fileReferences.length != 0) {
+          this.importFiles();
+        }
+      }
+    }
+  }
+
+  importFiles() {
+    this.fileReferences.forEach(fileReference => {
+      let excelTest = /.xlsx$/;
+      if (excelTest.test(fileReference.name)) {
+        const reader: FileReader = new FileReader();
+        reader.onload = (e: any) => {
+          /* read workbook */
```

Comment:
- Use let instead of var
- Remove commented ifelse block
- Comment (or rename) what we're matching in ln 100 regex